### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S *.po,./.git,./share/patchfiles,./libs,./msvc_extra_headers,./share/web_surfaces,*.patch  -L acount,ba,buss,busses,discreet,doubleclick,hsi,ontop,retrn,ro,scrollin,seh,siz,sord,sur,te,trough,ue`

### CODE REVIEW

File: libs/fluidsynth/src/fluid_rev.c [hunk 1]
```diff
@@ -435,7 +435,7 @@ static void set_mod_frequency(sinus_modulator *mod,
 
     a = (2 * FLUID_M_PI / 360) * phase;
 
-    mod->buffer2 = FLUID_SIN(a - w); /* y(n-1) = sin(-intial angle) */
```
Comment (x42) [inline libs/fluidsynth/src/fluid_rev.c:438]: This should go upstream to Fluidsynth. It will otherwise be overwritten when we sync libfluidsynth next time

Comment (luzpaz) [reply libs/fluidsynth/src/fluid_rev.c:438]: Submitted typo fix upstream in https://github.com/FluidSynth/fluidsynth/pull/1146  
Will revert this fix then

Comment (luzpaz) [reply libs/fluidsynth/src/fluid_rev.c]: squashed and rebased.

**Comment by luzpaz** [comment]
Updated branch, squashed and rebased

**Comment by pauldavisthefirst** [comment]
manually rebased and merged. thanks.

**Pull request closed without merging**